SessionMessagingKit/Sending & Receiving/Attachments/SignalAttachment.swift

```diff
@@ -20,6 +20,18 @@ public enum SignalAttachmentError: Error {
     case couldNotResizeImage
 }
 
+@objc
+public enum SignalAttachmentType: Int {
+    case text
+    case oversizeText
+    case image
+    case animatedImage
+    case video
+    case audio
+    case url
+    case unknown
+}
+
```

Comment:
Can we use the existed functions to check the attachments type?

Reply:
Yep I can change it back - the main reason I added this was because being able to switch on a type makes the code in `MediaMessageView` read a bit cleaner